Lesson2/admin.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib import admin
+
+# Register your models here.
+from .models import ClientModel, GoodModel, OrderModel, OrderListModel
+
+
+class ClientModelAdmin(admin.ModelAdmin):
+    list_display = ('name', 'email', 'phone', 'address', 'date_registration')
```

Comment:
Кортеж определяется запятой, не скобкой.
```suggestion
    list_display = 'name', 'email', 'phone', 'address', 'date_registration'
```